### PR TITLE
Authy integration

### PR DESCRIPTION
**Pull request by guga4ka**
## Description
Authy integration for Raycast. 

## Type of change

- New extension

## Screencast

output

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I checked that extension's icon has size of `512x512`

### CODE REVIEW

File: extensions/authy/src/component/OtpList.tsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                throw error;
+            }
+        }
+    }
+
+    useEffect(() => {
+        loadData()
+    }, [props.isLogin])
+
+    useInterval(() => {
```
Comment (guga4ka) [inline extensions/authy/src/component/OtpList.tsx:102]: Not sure that this is a good idea, but my goal was to show how long OTP will be valid. This leads to 2 re-renders each second.

File: extensions/authy/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Raycast Authy integration
+
+----------
+
+Warning: This extension is using Authy private api and may not compliant with the Terms of Service of Authy. Use at your own risk.
```
Comment (guga4ka) [inline extensions/authy/README.md:5]: Note about risk as requested here https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration

File: extensions/authy/package.json [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  "name": "authy",
+  "title": "Authy",
+  "description": "Raycast Authy integration",
+  "icon": "command-icon.png",
+  "author": "guga4ka",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "search-otp",
+      "title": "Search OTP",
```
Comment (pernielsentikaer) [inline extensions/authy/package.json:12]:
```suggestion
      "title": "Search OTP",
      "subtitle": "Authy",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @guga4ka 👋 

What an extension, very awesome — going to use this a lot myself 🔥 

A few comments:

- If the counter goes down to 1, and I go out and in again it, then the timer is starting from 30 again, but the code is still the same as earlier, will this cause a token mismatch?

- I have both Services and Apps, in Authy they all refresh every 30 second, in the extension it refreshed every 9 second — is this correct? 

- I suggest adding a ✅ in front of the HUD message to improve visibility

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @guga4ka 👋 

Thanks for the conversation in Slack, it was very giving 😄 

I suggest changing the title as the last thing, and then I think we're ready to go 💯

**Review by pernielsentikaer — APPROVED** [review]
Hi again @guga4ka 🔥 

Thanks for the update, I think we're good to go! 💪 Lets await final review before merge 😄 

Thanks for your corporation and conversations 💯

**Pull request merged**